### PR TITLE
add margin and padding pixel classes

### PR DESCRIPTION
**Pull request by hobbitronics**
Copied and modified the code for rem units to create margin and padding classes for 1-25 pixels.

### CODE REVIEW

File: components/global.scss [hunk 1]
```diff
@@ -67,6 +68,50 @@ $sides: (top: t, right: r, bottom: b, left: l);
   }
 }
 
+// UNITS: PIXELS!!
+$startPX: 0 !default;
+$endPX:  25 !default;
+
+// margin & padding m-1px through m-25px, p-1px through p-25px
+@each $property, $prop in $properties {
+  @for $i from $startPX through $endPX {
```
Comment (Schparky) [inline components/global.scss:77]: Seems like you're still dealing with some inconsistent indention. Appears to be two spaces here, and tabs on lines 85 and following. Have you looked at https://editorconfig.org/?

File: components/global.scss [hunk 2]
```diff
@@ -27,6 +27,7 @@ $end:  8 !default;
 $properties: (margin: m, padding: p);
 $sides: (top: t, right: r, bottom: b, left: l);
 
+//UNITS: REM !!
```
Comment (Schparky) [inline components/global.scss:30]:
```suggestion
// UNITS: REM !!
```

**Review by Schparky — APPROVED** [review]
Good idea to include this. My only comments are on formatting issues.

File: components/global.scss (hunk 1 above)
Comment (hobbitronics) [reply components/global.scss:77]: I use prettier but haven't set up a .prettierrc file on Ubuntu yet. Formatting command doesn't change anything so I probably need a config file first.

**Pull request merged**